src/savegame_json.cpp

```diff
@@ -225,7 +224,6 @@ void player::store(JsonOut &json) const
     json.member( "body_wetness", body_wetness );
 
     // breathing
-    json.member( "underwater", underwater );
     json.member( "oxygen", oxygen );
```

Comment:
Currently (in the merge build) I'm not showing an oxygen level being saved either in this point.  ???